### PR TITLE
updating fates default parameter file to api 36.1

### LINKED ISSUE #2783: Error in FatescoldLandUse testmod
### Brief summary of bug

Fails in:

ERS_D_Ld30.f45_f45_mg37.I2000Clm50FatesCruRsGs.derecho_intel.clm-FatesColdLandUse
ERS_D_Ld30.f45_f45_mg37.I2000Clm50FatesCruRsGs.izumi_nag.clm-FatesColdLandUse

### General bug information

**CTSM version you are using:** ctsm5.3.0

**Does this bug cause significantly incorrect results in the model's science?** No

**Configurations affected:** FatesColdLandUse test mod

### Details of bug

The shell_commands file in the testmod has -i-clm_demand for CLM_BLDNML_OPTS rather than --clm_demand

### Important output or errors that show the problem

preview_namelist shows:

```
Create namelist for component clm
   Calling /glade/work/erik/ctsm_worktrees/answer_changes/cime_config/buildnml
WARNING: CLM is starting up from a cold state
ERROR: Command /glade/work/erik/ctsm_worktrees/answer_changes/bld/build-namelist failed rc=255
out=
err=Unknown option: i-clm_demand
Unknown option: flanduse_timeseries
SYNOPSIS
```

### PR DESCRIPTION
**Pull request by rgknox**
### Description of changes

This set of changes simply updates the default parameter file pointer. This is a minor API change because, while no interface code has changed, the format of the new file has indeed changed.  

To do: 

- [x] A new default file must be loaded to the svn server
- [x] ".gitmodules" must be updated with the tag (will do so when fates-side tag is created)
- [x] The submodule pointer of the new fates tag must be added

### Specific notes

Contributors other than yourself, if any: @JessicaNeedham @XiulinGao @mpaiao @glemieux @jenniferholm

CTSM Issues Fixed (include github issue #): 

[Fates 1211](https://github.com/NGEET/fates/issues/1211)
Fixes #2783 

Are answers expected to change (and if so in what way)?

FATES answers will change.  FATES test suite has been run in a special parameter configurations previous to the PR to test B4Bness on fates side PRs.

Any User Interface Changes (namelist or namelist defaults changes)?  New fates parameter file default.

Does this create a need to change or add documentation? No

Testing performed, if any: will do regular and fates testing

### CODE REVIEW

**Comment by wwieder** [comment]
@ CLM meeting we discussed adding in the new arctic PFT changes to this pr too.  Is this going to happen before we merge?

**Comment by glemieux** [comment]
FATES-side testing ran into an issue (NGEET/fates#1254) last week, but we have a work around in place via https://github.com/ESCOMP/CTSM/pull/2700/commits/f2f75351febd4a413b0788dae256443fe5734510.  I'm rerunning tests today and should be able to get to the `aux_clm` tests as well.

**Comment by glemieux** [comment]
Testing on izumi is underway.

**Comment by glemieux** [comment]
Testing on derecho is complete.  Nearly all expected tests are B4B.  All the DIFFs against baseline are for the fates tests and are expected. 

There was a build error for a `nvhpc` compiler `FatesColdSatPhen` test that I hadn't seen before that I was able to clear up upon manually re-building the case:
```
317 NVFORTRAN-F-0004-Corrupt or Old Module file solarabsorbedtype.mod (/glade/u/home/glemieux/ctsm/components/cmeps/cime_config/../cesm/driver/esm.F90: 808)
318 NVFORTRAN/x86-64 Linux 24.3-0: compilation aborted
319 gmake: *** [/glade/u/home/glemieux/scratch/ctsm-tests/tests_1004-162240de/SMS.f45_f45_mg37.I2000Clm60FatesSpRsGs.derecho_nvhpc.clm-FatesColdSatPhen.GC.1004-162240de_nvh/Depends.nvhpc:149: esm.o] Error     2
```

The `ERS_Ly5_P128x1.f10_f10_mg37.IHistClm60BgcCrop.derecho_intel.clm-cropMonthOutput--clm-matrixcnOn_ignore_warnings` test failed because it hit the wall clock time.  I'll increase the time to 2 hours and resubmit.

**Comment by glemieux** [comment]
Regression testing on izumi is mostly complete.  I'm seeing two test hit issue #2800, so I'm resubmitting those:
```
FAIL ERI_D_Ld9_P48x1.f10_f10_mg37.I2000Clm50BgcCru.izumi_nag.clm-reduceOutput RUN time=292
FAIL ERP_D_Ld9.f10_f10_mg37.I1850Clm60BgcCrop.izumi_nag.clm-clm60cam7LndTuningModeLDust RUN time=360
```

I also ran into #2801.  Otherwise all expected non-fates tests are B4B.

**Comment by glemieux** [comment]
> Regression testing on izumi is mostly complete. I'm seeing two test hit issue #2800, so I'm resubmitting those:
> 
> ```
> FAIL ERI_D_Ld9_P48x1.f10_f10_mg37.I2000Clm50BgcCru.izumi_nag.clm-reduceOutput RUN time=292
> FAIL ERP_D_Ld9.f10_f10_mg37.I1850Clm60BgcCrop.izumi_nag.clm-clm60cam7LndTuningModeLDust RUN time=360
> ```

These two passed on resubmit.  Testing is complete.

File: doc/ChangeLog [hunk 1]
```diff
@@ -1,4 +1,91 @@
...
+ [PASS means all tests PASS; OK means tests PASS other than expected fails.]
+
+  regular tests (aux_clm: https://github.com/ESCOMP/CTSM/wiki/System-Testing-Guide#pre-merge-system-testing):
+
+    derecho ----- OK
+    izumi -------
+
+  fates tests: (give name of baseline if different from CTSM tagname, normally fates baselines are fates-<FATES TAG>-<CTSM TAG>)
+    derecho ----- OK
+    izumi -------
```
Comment (samsrabin) [inline doc/ChangeLog:59]: You did Izumi testing too, right? Would you mind marking those as OK as well?

File: doc/ChangeSum [hunk 2]
```diff
@@ -1,5 +1,7 @@
 Tag                   Who      Date  Summary
 ============================================================================================================================
+       ctsm5.3.003 multiple 10/03/2024 FATES default parameter file update
```
Comment (samsrabin) [inline doc/ChangeSum:3]: Please run `UpdateChangelog.pl -update`.

**Review by samsrabin — CHANGES_REQUESTED** [review]
Mostly good! Just two minor things in the ChangeLog/ChangeSum.

File: doc/ChangeSum (hunk 2 above)
Comment (glemieux) [reply doc/ChangeSum]: All set

File: doc/ChangeLog (hunk 1 above)
Comment (glemieux) [reply doc/ChangeLog]: Good catch.   Updated.

**Review by samsrabin — APPROVED** [review]

**Pull request merged**